### PR TITLE
Release 0.6.0 to master

### LINKED ISSUE #66: Release 0.6.0

### PR DESCRIPTION
**Pull request by rudolf-adamkovic**
Close #66

### CODE REVIEW

**Comment by andris-zalitis** [comment]
In review

File: AudioBar/Info.plist [hunk 1]
```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>0.5.2</string>
+	<string>0.5.3</string>
```
Comment (andris-zalitis) [inline AudioBar/Info.plist]: PR title says it's 0.6.0

**Review by andris-zalitis — CHANGES_REQUESTED** [review]
So which version is it going to be? :)

File: AudioBar/Info.plist (hunk 1 above)
Comment (rudolf-adamkovic) [reply AudioBar/Info.plist]: Oooooooops.

**Review by andris-zalitis — APPROVED** [review]
LGTM! :)

**Pull request merged**